src/material-experimental/mdc-slide-toggle/testing/slide-toggle-harness.ts

```diff
@@ -8,7 +8,7 @@
 
 import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
 import {coerceBooleanProperty} from '@angular/cdk/coercion';
-import {SlideToggleHarnessFilters} from './slide-toggle-harness-filters';
+import {SlideToggleHarnessFilters} from '@angular/material/slide-toggle/testing';
```

Comment:
Optional and mostly out of curiosity: why import through the module name here? The rule should be to only import through module name if it goes out of the current entry-point.

Reply:
Yeah I just missed that one because it didn't break anything, I'll fix it

Reply:
Oh nvm, this is the experimental MDC one. The filters have been moved out of experimental, so now we need to do this

Reply:
Oh my bad 🤦‍♂ I didn't realize